Formula/atlantis.rb

```diff
@@ -6,6 +6,11 @@ class Atlantis < Formula
   license "Apache-2.0"
   head "https://github.com/runatlantis/atlantis.git", branch: "master"
 
+  livecheck do
+    url :stable
+    regex(/^v?(\d+(?:\.\d+)+b?)$/i)
```

Comment:
```suggestion
    regex(/^v?(\d+(?:\.\d+)+)$/i)
```

This should use the standard regex for Git tags like 1.2.3/v1.2.3. The variation you're using looks like it's intended to match beta tags in a different formula/cask, as there aren't any tags in this repository using that format (and we don't want to match unstable releases anyway).